Smash_Combos.Core/Cqrs/Votes/PostComboVote/PostComboVoteRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Smash_Combos.Core.Cqrs.Votes.PostComboVote
+{
+    public class PostComboVoteRequest : IRequest<PostComboVoteResponse>
+    {
+        public int ComboId { get; set; }
+        public int CurrentUserId { get; set; }
+        public string UpOrDown { get; set; }
```

Comment:
This sort of thing should be a bool in the core, you can convert it from string in the mapping profile or controller action.